src/common/github.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * This file contains helper methods that create new GitHub requests
+ */
+
+const GITHUB_URL = 'https://api.github.com/repos/BEW111/testrepo/issues'; // this is a test github repo, change to actual repo once done
```

Comment:
Just leaving a comment here to make sure this gets updated. I think we should make this an env variable so we can have stage and local use an alternate repo